widgets/common/src/DynamicDropdown.tsx

```diff
@@ -138,12 +139,18 @@ export default function DynamicDropdown({
             toolbox
                 .getManager()
                 .doGet(fetchUrl, {
-                    params: {
-                        _search: searchQuery,
-                        _sort: valueProp,
-                        _size: pageSize,
-                        _offset: nextPage * pageSize
-                    }
+                    params: searchParam.reduce(
+                        (result: Record<string, any>, param) => {
```

Comment:
Minor:

We know that `searchQuery` is a string, so we can do:

```suggestion
                        (result: Record<string, string>, param) => {
```